internal/studio/studioHandlers.go

```diff
@@ -32,8 +32,7 @@ type StudioHandlers struct {
 	StudioURL      string
 	Server         *http.Server
 
-	mutex           sync.Mutex
-	mutexCondition  *sync.Cond
+	runningCond     *sync.Cond
 	running         bool
```

Comment:
```suggestion
	runningMutex     sync.Mutex
```

We don't need this state, it complicates it we can get my just fine with a simple mutex